### PR TITLE
feat(info): new features

### PR DESCRIPTION
**Pull request by pranshuchittora**
Continuation of => #833

Why new PR?
Let's start fresh.

### CODE REVIEW

File: packages/info/README.md [hunk 1]
```diff
@@ -9,22 +9,79 @@ This package returns a set of information related to the local environment.
...
 ### Node
 
 ```js
-const envinfo = require("@webpack-cli/info").default;
-envinfo();
+const info = require('@webpack-cli/info').default;
+
+async function wrapperFunc() {
+    await info({
+        /* Custom Config */
```
Comment (evenstensberg) [inline packages/info/README.md]: what's this for

Comment (pranshuchittora) [reply packages/info/README.md]: You mean `Custom Config`?
If you pass an object as per the defined schema then you can get info about those parts only, omitting the defaults.

Comment (evenstensberg) [reply packages/info/README.md]: Thanks for elaborating!

**Comment by evenstensberg** [comment]
@rishabh3112 could you take this out for a manual spin?

**Comment by pranshuchittora** [comment]
I am not able to find the `webpack-cli` in the `.bin` folder.

**Comment by rishabh3112** [comment]
Are you able to sort this out now? If not, I can send you my setup.

**Comment by evenstensberg** [comment]
It's at root in `cli.js` no big deal

**Pull request merged**